.github/workflows/base.yml

```diff
@@ -18,10 +18,22 @@ jobs:
             cuda_version: 11.8.0
             cuda_version_bnb: "118"
             pytorch: 2.0.0
+            axolotl_extras:
           - cuda: cu117
             cuda_version: 11.7.0
             cuda_version_bnb: "117"
             pytorch: 1.13.1
+            axolotl_extras:
+          - cuda: cu118
+            cuda_version: 11.8.0
+            cuda_version_bnb: "118"
+            pytorch: 2.0.0
+            axolotl_extras: gptq
+          - cuda: cu117
+            cuda_version: 11.7.0
+            cuda_version_bnb: "117"
+            pytorch: 1.13.1
+            axolotl_extras: gptq
```

Comment:
Do we need to think about python>3.9 or let it be only 3.9 for now? It may be a waste of resources if we support too many kinds.

I also remember getting an image depreciation warning with cuda:117 and torch1.13.1. Not sure if the image that you base on will suddenly be deleted later

Reply:
yeah, the image count could really explode fast. maybe only gptq on a single group of versions and python?

Reply:
yeah, would like to add python 3.10 too at some point, maybe just for the newer PyTorch